libs/cart/src/drivers/interfaces/cart-payment-methods-service.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { InjectionToken } from '@angular/core';
+
+export interface DaffCartPaymentMethodsServiceInterface {
+	list(): any[];
```

Comment:
Do you think this should be using generics for the return type? I know the payment method will probably be an enum or something in the future but the service interfaces should be as specific as we can make them. I feel like `T extends string` is better than `any`.

Reply:
`T extends string` doesn't make sense to me. Isn't that meaningless?

Reply:
I didn't do a generic here because I don't really have a good understanding of what the type may be at the moment.

Reply:
@lderrickable its future-proofing for when it is replaced by a real type. Not really necessary though and might make it more confusing to read.

Reply:
I've addressed this, LMK what you think.

Reply:
lgtm